### PR TITLE
migration for gbloc additions

### PR DESCRIPTION
**Pull request by kilbergr**
## Description

This is a PR to create a gbloc field on the transportation_offices table and then import information from [this spreadsheet](https://docs.google.com/spreadsheets/d/1icaq6KFG44fJJZ1KichIk3Uuk2h2Q6njW3nkFifaqYo/edit#gid=0) into the table, matched according to the offices' names.

## Reviewer Notes

Since this migration is happening now, future imports of transportation offices with similar names should not cause GBLOCs to be incorrectly assigned. Please alert if my understanding is incomplete.
These match on snippets. Some transportation office names we have in the db map to multiple GBLOCs. I have made best guesses. Please alert if you have a better idea how to do.

## Code Review Verification Steps

* [x] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [x] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/159386663) for this change

### CODE REVIEW

**Comment by tinyels** [comment]
Was going to approve, but I don't have access to spreadsheet for confirmation.

File: migrations/20180802003208_import_gbloc_on_transportation_offices.up.sql [hunk 1]
```diff
@@ -0,0 +1,166 @@
+--
+-- Data for Name: transportation_offices; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+UPDATE public.transportation_offices SET gbloc='EBAK' where name LIKE '%Army Sustainment Command%';
```
Comment (ntwyman) [inline migrations/20180802003208_import_gbloc_on_transportation_offices.up.sql]: Using LIKE here is a) slow b) susceptible to errors if an expression matches more than one entry.

At the very least you should include a 'LIMIT 1' in each statement to contain any accidents. Ideally tho the where clauses should be WHERE id = 'xxx'.

The Primary Keys (UUIDs) for the transportation_offices are included in the migrations which create them so you can legitimately use those directly for this,

File: migrations/20180801223824_add_gbloc_transportation_offices.up.fizz [hunk 2]
```diff
@@ -0,0 +1 @@
+add_column("transportation_offices", "gbloc", "string", {"null": true})
```
[the remark this answers is not in the document]
Comment (kilbergr) [reply migrations/20180801223824_add_gbloc_transportation_offices.up.fizz]: There are currently a number of transportation offices in the db that don't have gblocs. What should do about that?

File: swagger/internal.yaml [hunk 3]
```diff
@@ -92,6 +92,10 @@ definitions:
           format: telephone
           pattern: '^[2-9]\d{2}-\d{3}-\d{4}$'
           example: 212-555-5555
+      gbloc:
+        type: string
+        format: string
```
Comment (ntwyman) [inline swagger/internal.yaml]: might add a pattern to this  '[A-Z]{4}'

**Pull request merged**